README.md

```diff
@@ -306,6 +307,9 @@ To illustrate, the typical visit loop would look like, given a `MyVisitor: ListV
 - `<Node as Drive>::drive_inner(ListVisitorWrapper(v))`
 - calls `<MyVisitor as ListVisitor>::visit(v, &x.field)` on each field of `x`
 
-Note: the `visitable_group` interface makes it possible to write composable visitor wrappers
-that provide reusable functionality. For an example, see
+Notes: 
+ - the `visitable_group` interface makes it possible to write composable visitor
+wrappers that provide reusable functionality. For an example, see
 [`derive_generic_visitor/tests/visitable_group_wrapper.rs`].
+ - it is also able to generate a infaillable trait interface, e.g. a trait where the `visit_$ty`, `enter_$ty`, etc. methods return `()` instead of `ControlFlow<Infaillible>`.
```

Comment:
Could you say how? Also I think this is not the right place for this info, it should be next to where we list the other options.

Also `enter_ty`/`exit_ty` are always infallible, only `visit_ty` is fallible.

Reply:
Sure, I actually pushed d0f31c13ea1043b28936d74c4694a93d5f7d0f00 that adds a more comprehensive list of options specifically for the `visitable_group` macro, wdyt?